src/command/explain.rs

```diff
@@ -0,0 +1,35 @@
+use crate::command::RoverStdout;
+use crate::Result;
+use crate::error::metadata::code::Code;
+use structopt::StructOpt;
+use serde::Serialize;
+use std::convert::TryFrom;
+use ansi_term::Colour::{Red};
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Explain {
+    /// The full error code. For example, E020
+    #[structopt(name = "CODE")]
+    code: String,
+
+    #[structopt(long="all", hidden=true)]
+    all: bool
```

Comment:
mebbe a special cased `EALL` for all errors would be better suited instead of combo required positional and `--all` flag (esp. if we do my above suggestion requiring structopt to do the parsing of the code)